test/categorical_algebra/DPO.jl

```diff
@@ -0,0 +1,725 @@
+module DPOTest
+using Test
+using Catlab.CategoricalAlgebra.DPO
```

Comment:
With a few exception, the top-level module re-export their submodules, you can replace most of these imports with:

```julia
using Catlab, Catlab.Theories, Catlab.CategoricalAlgebra
using Catlab.Graphics, Catlab.WiringDiagrams
```